docs/tutorials/tut1.rst

```diff
@@ -41,16 +41,15 @@ Let's create a password and provide it to PostgREST. Think of a nice long one, o
     export LC_CTYPE=C
 
     # read random bytes and keep only alphanumerics
-    < /dev/urandom tr -dc A-Za-z0-9 | head -c32
+    echo "jwt-secret = \"$(< /dev/urandom tr -dc A-Za-z0-9 | head -c32)\"" > tutorial.conf
```

Comment:
`>` is the replace command. Changing it to `>>` (append) instead. Otherwise it'll override the tutorial.conf from tutorial 0.

```suggestion
    echo "jwt-secret = \"$(< /dev/urandom tr -dc A-Za-z0-9 | head -c32)\"" >> tutorial.conf
```